haystack/cli/README.md

```diff
@@ -13,32 +13,42 @@ This is the repository where we keep the code for the Haystack CLI.
 
 To contribute to the tutorials please check out our [Contributing Guidelines](./Contributing.md)
 
-## Available commands
+## Available CLI commands
 
 ### `haystack prompt fetch`
 
-```
-Usage: haystack prompt fetch [OPTIONS] [PROMPT_NAME]...
+**Usage:**
 
-  Downloads a prompt from the official Haystack PromptHub and saves it locally
-  to ease use in environments with no network.
+```
+haystack prompt fetch [OPTIONS] [PROMPT_NAME]
+```
 
-  PROMPT_NAME can be specified multiple times.
+Download a prompt from the official [Haystack PromptHub](https://prompthub.deepset.ai/) and save it locally
+for easier use in environments with no network.
 
-  PROMPTHUB_CACHE_PATH environment variable can be set to change the default
-  folder in which the prompts will be saved in.
+- PROMPT_NAME can be specified multiple times.
 
-  If a custom PROMPTHUB_CACHE_PATH is used, remember to also use it for
-  Haystack invocations.
+- PROMPTHUB_CACHE_PATH environment variable can be set to change the default
+folder in which the prompts will be saved in. You can find the default cache path on your machine by running the following code:
```

Comment:
I find this presentation slightly confusing. `PROMPT_NAME` and `PROMPTHUB_CACHE_PATH` are very different:

- `PROMPT_NAME` is just a placeholder that comes from the shell's help string (see line 23).
- `PROMPTHUB_CACHE_PATH` is an actual env var.

Putting them on the same level seems to strangely imply that they're both env vars.

My suggestion would be to remove the list and just use two lines, I think that already helps. On top of that, maybe line 29 can be changes into simply `You can specify multiple prompts to fetch.` or something on these lines, without mentioning `PROMPT_NAME` at all: the average developer should be able to figure it out (I hope :smile:)

Reply:
I like both suggestions, added them :)